### PR TITLE
 Develop the Newcomers Page First Timers Issue section

### LINKED ISSUE #252: Develop the Newcomers Page First Timers Issue section
## Description
As a contributor,
I need create a section on the newcomers page for first timer issues.

### Update [Required]
- [ ] newcomers.component.html
- [ ] newcomers.component.ts
- [ ] newcomers.component.scss

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
24 hours 

@ancyp @Janiceilene @bethanyr Can you please assign me this issue? Thanks! :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
As a contributor,
I need create a section on the newcomers page for first timer issues.

Fixes #252 

TIme Taken:
24 hours (6 hours research + 2 hours implementation + 2 hours resolving merge conflicts on my local branch - all my PRs on the newcomers page is leading to a lot of conflicts + 3 hours reading about the iframe issue, Bethany mentioned about issues not being clickable - blank frame opening up+ 7 hours implementing a modular implementation, getting error at Array implementation, getting stuck at the unsafe URL error, looked up and tried solutions (mentioned on slack: https://systers-opensource.slack.com/archives/C99N86T43/p1532619465000180) + 3 hours - implemented less modular but working code using tabsets + 1 hour - added all projects info acc to their technology stack manually)

### Type of Change:
- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation
  
**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Ran it locally

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
  
**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by bethanyr** [comment]
When I click on a newcomer issue, then I just see a blank area in the iframe.

**Comment by BaaniLeen** [comment]
@bethanyr It works properly and opens up in a new tab, but yes, it cannot open the link within the iframe and gives this error: ```Refused to display 'https://github.com/systers/systers.github.io/issues/243' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'none'".```
Looking up about it.

**Comment by bethanyr** [comment]
@BaaniLeen - now there's conflicts in this PR

**Comment by BaaniLeen** [comment]
@bethanyr Resolved the merge conflict :)

**Comment by ancyp** [comment]
There needs to be some indication that it needs to be opened in a new page.

File: src/app/newcomers/newcomers.component.html [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+        </p>
+        <p>
+          We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation,
+          ability, ethnicity, socioeconomic status, work experience, or religion (or lack thereof).
+        </p>
+        <p class="mb-5">
+          We believe in inclusivity, diversity, and transparency. We value open source citizenship and collaborative involvement.
+        </p>
+      </div>
+      <div class="row justify-content-center mb-3">
```
Comment (bethanyr) [inline src/app/newcomers/newcomers.component.html]: I thought you had a previous PR that removed the map from the newcomer's page? Or was it a different page?

File: src/app/newcomers/newcomers.component.html [hunk 2]
```diff
@@ -153,4 +168,113 @@ <h5 class="card-title">{{det.name}}</h5>
...
     </div>
+    <div class="container box logos mx-auto d-block mt-5 mb-5">
+      <h1 class="text-center mt-5 mb-5">First-Timers-Only Issues</h1>
+      <ngb-tabset [justify]="currentJustify">
+        <ngb-tab title="Web Dev - Python">
+          <ng-template ngbTabContent>
+            <!-- Peace Corps Projects Note-->
+            <div class="row text-center">
+              <h1 class="mx-auto mt-5 mb-5">Sysbot</h1>
+              <iframe class="iframe-style" src="https://publiclab.github.io/community-toolbox/examples/embed.html#o=systers&r=sysbot"></iframe>
```
Comment (bethanyr) [inline src/app/newcomers/newcomers.component.html]: are you planning to only show first-timer issues or all issues for a repo?

File: src/app/newcomers/newcomers.component.html (hunk 1 above)
Comment (BaaniLeen) [reply src/app/newcomers/newcomers.component.html]: @bethanyr Done. :)

**Comment by ancyp** [comment]
I don't think we decided on a solution to the empty iframe in the meeting. Is there a way to do it other than iframes?
If not can we make it a separate issue and merge this?

**Comment by BaaniLeen** [comment]
@ancyp This was the discussion wherein we decided to use iframes. I would suggest we close this issue for now, and create a separate issue for the empty iframe thing. What do you say, @bethanyr @Janiceilene ? :)

**Comment by bethanyr** [comment]
Let's go ahead and create another issue to implement without iframes so you can control how the links open and we can merge this item in for now.

**Pull request merged**